uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -427,6 +427,20 @@ function hrleaveandabsences_civicrm_validateForm($formName, &$fields, &$files, &
   }
 }
 
+/**
+ * Implementation of hook_civicrm_pageRun
+ */
+function hrleaveandabsences_civicrm_pageRun(&$page) {
+  if ($page instanceof CRM_HRLeaveAndAbsences_Page_AbsenceTab) {
```

Comment:
Instead of using the hook, you can use the page itself (in its `run` method) to push the permissions